libkbfs/folder_branch_ops.go

```diff
@@ -2864,8 +2864,10 @@ func (fbo *folderBranchOps) checkNewDirSize(ctx context.Context,
 	// Just an approximation since it doesn't include the size of the
 	// directory entry itself, but that's ok -- at worst it'll be an
 	// off-by-one-entry error, and since there's a maximum name length
-	// we can't get in too much trouble.
-	if currSize+uint64(len(newName)) > fbo.config.MaxDirBytes() {
+	// we can't get in too much trouble. TODO(KBFS-3306): remove this
+	// check.
+	if dirPath.tailPointer().DirectType == DirectBlock &&
```

Comment:
Could you explain why this check is needed now and will be be removed when we enable block splitting (KBFS-3306)?

Reply:
Yes: https://github.com/keybase/kbfs/blob/master/libkbfs/prev_revisions.go#L22-L26

(It's 5.)

Reply:
Thanks; this seem to answer my question in the first comment instead of this though?

Reply:
Oops sorry, weird.  This check is needed temporarily now that we estimate the block size for indirect directories using `MaxBlockSizeBytesDefault`.  For tests that set the block size to something small, like 20 bytes, they will make indirect directories, but the size of the directories will appear to be very large because of that size heuristic.  So this check causes unneeded errors for those tests.

In KBFS-3306 we will remove the directory size check completely, so this whole function will go away.

Reply:
I see; thanks for explaining!